source/code/scxsystemlib/disk/diskdepend.cpp

```diff
@@ -681,7 +681,7 @@ namespace SCXSystemLib
             L"tmpfs",
             L"udfs", L"usbfs",
 #if defined(linux)
-            L"udev",
+            L"udev", L"efivarfs", L"fuse.lxcfs",
```

Comment:
We like to keep these more or less alphabetical to make it easier to find a specific entry in the long list. Not mandatory, but certainly nice to do, and makes code easier to maintain somewhat.